root/application/src/main/java/at/fhv/itb5c/application/ApplicationFacade.java

```diff
@@ -248,4 +248,40 @@ private boolean hasRole(String sessionId, UserRole... roles) {
 		}
 		return false;
 	}
+	
+	public TournamentDTO createTournament(String sessionId, DepartmentDTO dept) {
+		if (hasRole(sessionId, UserRole.Admin) && isDepartmentHead(_sessionManager.getUserId(sessionId), dept)) {
+			return ConverterTournamentDTO.toDTO(new Tournament());
+		}
+		return null;
+	}
+	
+	public TournamentDTO saveTournament(String sessionId, TournamentDTO tournament, DepartmentDTO dept) {
+		if (hasRole(sessionId, UserRole.Admin) && isDepartmentHead(_sessionManager.getUserId(sessionId), dept)) {
```

Comment:
at the moment to save a department we need to be admin AND head of department.. use `||` instead of `&&` so that EITHER HoD or Admin can save tournaments